zbus_names/src/bus_name.rs

```diff
@@ -359,9 +359,15 @@ impl Borrow<str> for OwnedBusName {
     }
 }
 
+impl Debug for OwnedBusName {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        Debug::fmt(&BusName::from(self), f)
+    }
+}
+
 impl Display for OwnedBusName {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        BusName::from(self).fmt(f)
+        Display::fmt(&BusName::from(self), f)
```

Comment:
> Before:
> 
> Owned*Name(*Name("foo"))
> 
> After:
> 
> *Name("foo")

Hm.. maybe this should be `Owned*Name("foo")`?

Reply:
I don't have strong opinion about it, but I mirrored it on how the ownership information on `Str` is ignored

Reply:
I think it may also be mislead some people that a `*Name` is not owned because it doesn't have a `Owned` prefix, but it actually have `'static` lifetime

Reply:
Good point but I don't think that's too relevant an info here. If the lifetime info is important here, we should show that for  `Str` and all its wrappers (which can be done separately later). The point is that we should show the name of the actual Type here.

Reply:
Pushed changes that modifies the second commit to show the name of the actual type